web-frontend/src/pages/AddAccount/AddAccount.tsx

```diff
@@ -0,0 +1,37 @@
+import LandingPage from '../../layouts/LandingPage/LandingPage';
+import { FiArrowRight } from 'react-icons/fi';
+import { Button } from '@massalabs/react-ui-kit/src/components/Button/Button';
+
+export default function AddAccount() {
+  const topButtonArgs = {
+    onClick: () => {
+      console.log('clicked');
+    },
+    posIcon: <FiArrowRight />,
+  };
```

Comment:
@mazmassa why it is better to declare the props in a const here rather than directly where you use the component.

IMO it could be hard to read the code if I have to scroll up to the const declaration while reading the HTML bellow.

Reply:
Agree!

This may is useful in tests to not set in each one. However in the implementation it would worth put directly in the component, indeed.

Reply:
However, since this is already merged... maybe a scaling task to update then.

Reply:
okay sounds good i'll take note in my future integrations :) @Thykof  @mazmassa

Reply:
Nice recommendation! 👍🏻 
I will create à PR to update this on the ui-kit for my Button component.